src/LiveDevelopment/LiveDevelopment.js

```diff
@@ -1369,7 +1369,10 @@ define(function LiveDevelopment(require, exports, module) {
             // wait for server (StaticServer, Base URL or file:)
             prepareServerPromise
                 .done(function () {
-                    WebSocketTransport.createWebSocketServer(PreferencesManager.get("livedev.wsPort"));
+                    var wsPort = PreferencesManager.get("livedev.wsPort");
```

Comment:
Any reason not to use a different pref instead, such as `livedef.enableReverseInspect`

Reply:
@petetnt IMO, that might confuse a lot of users. Instead if an invalid port (0) is given by the user, it's implicit to assume that reverse inspect is not required.

Reply:
My 2 cents is that average user doesn't even know what a `port` is, even without considering what might be an invalid port and what isn't 🤔

Reply:
I am a pendulum now 😃 . I had asked the same initially to @saurabh95 but somehow got convinced with the explanation. But now I am convinced by your explanation again so would ask the author to include a new preference. @saurabh95 over to you...

Reply:
Yeah, for a user who is not aware of ports, it would be a difficult way to disable reverse inspect. I will  add a new preference and will update this PR only.

Reply:
👍